tests/fixtures/organisations.py

```diff
@@ -0,0 +1,628 @@
+    flush_index(PatronTypesSearch.Meta.index)
+    return ptty
+
+
+@pytest.fixture(scope="module")
+def patron_type_grown_sion_data(data):
+    """Load grown patron type of sion."""
+    return deepcopy(data.get('ptty4'))
+
+
+@pytest.fixture(scope="module")
+def patron_type_grown_sion(
+        app, org_sion, patron_type_grown_sion_data):
+    """Crate grown patron type of sion."""
+    ptty = PatronType.create(
+        data=patron_type_grown_sion_data,
+        delete_pid=False,
+        dbcommit=True,
+        reindex=True)
+    flush_index(PatronTypesSearch.Meta.index)
+    return ptty
+
+
+@pytest.fixture(scope="function")
+def circ_policy_data_tmp(data):
+    """Load default circ policy for organisation martigny as temp."""
+    return deepcopy(data.get('cipo1'))
+
+
+@pytest.fixture(scope="module")
+def circ_policy_default_martigny_data(data):
+    """Load default circ policy for organisation martigny."""
+    return deepcopy(data.get('cipo1'))
+
+
+@pytest.fixture(scope="module")
+def circ_policy_default_martigny(
+        app, org_martigny, circ_policy_default_martigny_data):
+    """Create default circ policy for organisation martigny."""
+    cipo = CircPolicy.create(
+        data=circ_policy_default_martigny_data,
+        delete_pid=False,
+        dbcommit=True,
+        reindex=True)
+    flush_index(CircPoliciesSearch.Meta.index)
+    return cipo
+
+
+@pytest.fixture(scope="module")
+def circ_policy_default_sion_data(data):
+    """Load default circ policy for organisation sion."""
+    return deepcopy(data.get('cipo4'))
+
+
+@pytest.fixture(scope="module")
+def circ_policy_default_sion(
+        app, org_sion, circ_policy_default_sion_data):
+    """Create default circ policy for organisation sion."""
+    cipo = CircPolicy.create(
+        data=circ_policy_default_sion_data,
+        delete_pid=False,
+        dbcommit=True,
+        reindex=True)
+    flush_index(CircPoliciesSearch.Meta.index)
+    return cipo
+
+
+@pytest.fixture(scope="module")
+def circ_policy_short_martigny_data(data):
+    """Load short circ policy for organisation martigny."""
+    return deepcopy(data.get('cipo2'))
+
+
+@pytest.fixture(scope="module")
+def circ_policy_short_martigny(
+        app,
+        patron_type_children_martigny,
+        patron_type_adults_martigny,
+        item_type_standard_martigny,
+        item_type_specific_martigny,
+        circ_policy_short_martigny_data):
+    """Create short circ policy for organisation martigny."""
+    cipo = CircPolicy.create(
+        data=circ_policy_short_martigny_data,
+        delete_pid=False,
+        dbcommit=True,
+        reindex=True)
+    flush_index(CircPoliciesSearch.Meta.index)
+    return cipo
+
+
+@pytest.fixture(scope="module")
+def circ_policy_temp_martigny_data(data):
+    """Load temporary circ policy for organisation martigny.
+
+    library martigny-ville.
+    """
+    return deepcopy(data.get('cipo3'))
+
+
+@pytest.fixture(scope="module")
+def circ_policy_temp_martigny(
```

Comment:
tmp or temp? at the end?

Reply:
here is a temporary circ policy .. temp is correct